packages/api/src/services/express.ts

```diff
@@ -20,8 +20,6 @@ export default async (config: Config, actions: Actions) => {
   app.use(bodyParser.json({ limit: "1mb" }));
   app.use(bodyParser.urlencoded({ extended: true }));
 
-  app.options("*", cors());
```

Comment:
@daywiss I had to fix this to fix a TS compile error similar to [this issue](https://stackoverflow.com/questions/59167258/no-overload-matches-this-call-in-cors-typescript).

The solution was found when someone pointed out the correct way to use `cors` is like so.

[cors docs](https://expressjs.com/en/resources/middleware/cors.html)

Reply:
Thanks nick!